web-lib-funcs.pl

```diff
@@ -15,6 +15,7 @@ =head1 web-lib-funcs.pl
 ##use warnings;
 use Socket;
 use POSIX;
+use IO::Handle;
```

Comment:
Is this module always available?

Reply:
Yes.

Reply:
Ok and just including this allows us to call sync() on any file handle?

Reply:
> Ok and just including this allows us to call sync() on any file handle?

Yes, in modern Perl, **lexical filehandles** created with `open` are _automatically_ blessed into a class that inherits from `IO::Handle`...

Reply:
I see!